gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -53,6 +53,7 @@ class _AccumulatedData:
     grand_totals_headers: List[Optional[List[Dict[str, _DataHeaders]]]] = field(
         init=False, factory=lambda: [None, None]
     )
+    total_of_grant_totals_processed: bool = field(init=False, factory=lambda: False)
```

Comment:
Nitpick: instead of `factory=lambda: False` you should be able to do `default=False`.

Reply:
fixed